prototypes/entity/e_advmachinery.lua

```diff
@@ -213,4 +213,4 @@ data:extend(
 		allowed_effects = {"consumption", "speed", "pollution"},
 	},	
 	
-})
```

Comment:
I have no idea what git-hub is doing here. There is no line break in my file. Even editing this on the web causes this. Its harmless at lest, just annoying.

Reply:
Thanks josh :)